randomname/util.py

```diff
@@ -202,7 +205,7 @@ def as_valid_path(name, required=False):
 
 def prefix(pre, xs):
     '''Prefix all items with a path prefix.'''
-    return [os.path.join(pre, x.lstrip('/')) for x in as_multiple(xs)]
+    return [f"{pre}/{x.lstrip('/')}" for x in as_multiple(xs)]
```

Comment:
this means that users on both Windows / Unix use `/` for their names. This avoids that the library behavior changes based on the OS which I found weird (up to you though)

Reply:
I guess no one really cares about python 3.5 so I guess f strings are not a compatability concern anymore haha

Reply:
Security support for 3.5 already ended (3.6 is dropped this winter) so I'd drop it 👍🏻